packages/esbuild/test/css/main.ts

```diff
@@ -1,3 +1,5 @@
 import { dep } from "./dep";
 
 dep();
+
+console.log(process.cwd())
```

Comment:
intentional?

Reply:
Perhaps I'll split this into another test. The input has a shim defined for `process.cwd` so it should result in an empty string, the test asserts that the shim is picked up correctly.

Reply:
Moved to the define test, make a little more sense there